package.json

```diff
@@ -100,6 +100,7 @@
       "!lib/**/{__tests__,testUtils}/**/*.{js,mjs}"
     ],
     "coverageDirectory": "./.coverage/",
+    "coverageProvider": "v8",
```

Comment:
minor

You didn't remove this line after opening #7025.

Reply:
Leaving `coverageProvider` is intentional to make CI successful by `coverageProvider: v8`.

I wait for merging #7025, and I'll fix a conflict that may be caused by the merge.